NEWS.md

```diff
@@ -1,5 +1,42 @@
 <!-- NEWS.md is maintained by https://cynkra.github.io/fledge, do not edit -->
 
+## fix
+Fix conventional commit regex (#329)
+
+
+## feat
+improve bump_version() (error) messages (#328)
+
+Co-authored-by: maelle <maelle@users.noreply.github.com>
```

Comment:
To be removed?

Reply:
I don't understand why these appear now here. Let's discuss.

Reply:
Because they're part of the commit message for now.

![image](https://user-images.githubusercontent.com/8360597/169526659-ce65cfac-1411-4d92-9d72-b23488629a2f.png)